openpype/hosts/maya/plugins/publish/validate_gltf_textures_names.py

```diff
@@ -0,0 +1,118 @@
+from maya import cmds
+
+import pyblish.api
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidateGLTFTexturesNames(pyblish.api.InstancePlugin):
+    """
+    Validate if the asset uses StingrayPBS material before conversion
+    of the GLSL Shader
+    Validate if the names of GLTF Textures follow
+    the packed ORM/ORMS standard.
+    The texture naming conventions follows the UE5-style-guides:
+    https://github.com/Allar/ue5-style-guide#anc-textures-packing
+    ORM: Occulsion Roughness Metallic
+    ORMS: Occulsion Roughness Metallic Specular
+    Texture Naming Style:
+    Albedo/Diffuse: {Name}_D.{imageExtension} or
```

Comment:
Is there a reason the `{Name}` key is uppercase?

Also, I feel `{imageExtension}` is overly verbose and `{ext}` or `{extension}` would do just as fine.

Reply:
> Is there a reason the `{Name}` key is uppercase?

This is probably because of PascalCase required by th [UE5 naming convention](https://github.com/Allar/ue5-style-guide#211-always-use-pascalcase)